src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-form/silence-form.component.ts

```diff
@@ -334,4 +335,18 @@ export class SilenceFormComponent {
     }
     return `${action} ${this.resource} ${id}`;
   }
+
+  fetchData() {
+    if (history.state.title) {
+      this.setMatcher({
+        name: 'alertname',
+        value: history.state.title.split(' ')[0],
+        isRegex: false
+      });
+      this.createForm();
+      this.form.get('comment').setValue('Silence created successfully');
+      this.setupDates();
+      this.submit();
+    }
```

Comment:
This piece of data should probably be reused from the "Create Silence" in the Active Alert page. Any chance to do that?